src/databricks/labs/ucx/hive_metastore/table_migrate.py

```diff
@@ -503,3 +510,174 @@ def _sql_alter_from(self, table: Table, target_table_key: str, ws_id: int):
     def _is_migrated(self, schema: str, table: str) -> bool:
         index = self._migration_status_refresher.index()
         return index.is_migrated(schema, table)
+
+
+class ACLMigrator:
+    def __init__(
+        self,
+        tables_crawler: TablesCrawler,
+        grant_crawler: GrantsCrawler,
+        workspace_info: WorkspaceInfo,
+        backend: SqlBackend,
+        group_manager: GroupManager,
+        migration_status_refresher: MigrationStatusRefresher,
+        principal_acl: PrincipalACL,
+    ):
+        self._tc = tables_crawler
+        self._gc = grant_crawler
+        self._backend = backend
+        self._workspace_info = workspace_info
+        self._group = group_manager
```

Comment:
```suggestion
        self._tables_crawler = tables_crawler
        self._grants_crawler = grant_crawler
        self._sql_backend = backend
        self._workspace_info = workspace_info
        self._group_manager = group_manager
```

code is read more often than it's written. please consistently name all properties everywhere

Reply:
Addressed